### PR TITLE
RTP streams plot

### PR DESCRIPTION
**Pull request by Falon452**

### CODE REVIEW

File: client/src/app/gui/rtp_streams_plot.rs [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+                    self.settings_x_axis = RtpTimestamp;
+                    self.requires_reset = true
+                }
+                if ui
+                    .radio(is_sequence_number, "X axis is sequence number")
+                    .clicked()
+                {
+                    self.settings_x_axis = SequenceNumer;
+                    self.requires_reset = true
+                }
```
Comment (LVala) [inline client/src/app/gui/rtp_streams_plot.rs:65]: Add `all` functions to `SettingsXAxis` that returns vector of all of the enum options and do the `ui.radio(...)` in a loop over the possible options.

**Review by LVala — CHANGES_REQUESTED** [review]
CI is not passing

File: client/src/app/gui/rtp_streams_plot.rs (hunk 1 above)
Comment (Falon452) [reply client/src/app/gui/rtp_streams_plot.rs]: image
does not work, i would prefer leave it as it is

File: client/src/app/gui/rtp_streams_plot.rs [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+use crate::streams::RefStreams;
+
+#[derive(Debug)]
+pub enum SettingsXAxis {
+    RtpTimestamp,
+    RawTimestamp,
+    SequenceNumer,
+}
+
+#[cfg_attr(feature = "serde", derive(serde::Deserialize, serde::Serialize))]
```
[the remark this answers is not in the document]
Comment (Falon452) [reply client/src/app/gui/rtp_streams_plot.rs]: deleted

**Review by LVala — APPROVED** [review]

**Pull request merged**